_posts/2023-08-23-dashboards-usability-snapshot.md

```diff
@@ -0,0 +1,79 @@
+
+The sessions were conducted from May 16--June 22, 2023. Producer sessions were conducted first. After successful pilots (n=2) were conducted May 16—19, an additional 13 Producers sessions were conducted May 24--June 2, 2023. Consumer sessions were conducted after successful pilots (n=2) and were conducted June 7--22, 2023. Sample size was 30 participants (15 Producers and 15 Consumers), including four pilots split evenly between the user groups. Sessions were organized as follows: 5-minute introduction and pre-brief; 5-minute overview of participants’ use cases; 45-minute period for testing tasks; and 5-minute post-test discussion and debrief.
+
+We asked Producers and Consumers to do the following tasks:
+
+### Producer tasks
+
+The Producer participants were asked to do the following tasks:
+
+1. Create an index pattern.
+2. Create a saved search in Discover and select a few columns from the provided dataset.
+3. Create a visualization using any of the visualization tools in Dashboards, including the new experimental VisBuilder app.
+4. Create a dashboard and share it.
+5. Analyze an existing dashboard and identify an anomaly in the synthetic dataset.
+6. Generate a report.
+
+### Consumer tasks
+
+The Consumer participants were asked to do the following tasks:
+
+1. Analyze an existing dashboard and identify an anomaly in a sample dataset.
+2. Clone a dashboard to further analyze the dashboard data.
+3. Find the pages responsible for the most 404 error codes in a sample dataset.
+4. Find a specific log record.
+5. Generate a report.
+
+## Key learnings
+
+Producer and Consumer participants were selected for their deep expertise in dashboard processes and behaviors. Producers and Consumers have different roles: Producers make the dashboards and share them, while Consumers use the dashboards that are shared. 
+
+1. **Dashboard Producers:** Producers experienced a success rate of approximately 33% and an additional 38% with assistance. They had a failure rate of approximately 30%. The most successful task was to create a view for log data, and the least successful task was to create an index pattern. Dashboard producers did not understand how to select the desired index in order to create a new index. Many expected to add indexes by clicking on the individual index name. Users tried to click on ‘Index pattern’ displayed next to the index names. Other complexities such as using an asterisk to select multiple indexes were not intuitive to the user, further creating stumbling blocks to this critical first step in setting up dashboards. 
```

Comment:
This list should be preceded by a brief introductory sentence ending in a colon.

Reply:
Added intro sentence "Let's take a closer look at these two roles: ..."